corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -52,6 +52,8 @@ def _run_upload(domain, workbook, replace=False, task=None):
     def process_table(table, new_table):
         def process_row(row, new_row):
             update_progress(table)
+            if skip_orm:
```

Comment:
The `skip_orm` parameter references the feature flag that enables the "fast" fixture uploader.

https://github.com/dimagi/commcare-hq/blob/033796fc2867f0c4f4f23d1a67bbcd429b00b471/corehq/toggles/__init__.py#L1806-L1808

That language is now less accurate than it was before, and once Lookup Tables are migrated to SQL that language will be even less meaningful, but changing it seems out of scope for the migration. Rather than change it, I would prefer to remove that feature flag since I think the benefits will be negligible with the new uploader. I will wait for the results of performance testing to decide how to proceed.

Reply:
I am so curious what that person meant, but agree it is definitely not worth changing.